codegen/render/template_renderer.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/gobuffalo/packr"
+)
+
+// map of template files to the file they render to
+type inputTemplates map[string]OutFile
+
+func (r *inputTemplates) add(t inputTemplates) {
+	if *r == nil {
```

Comment:
`if r == nil`? possible panic

Reply:
yes `r` should never be nil but `*r` can be nil since maps can be nil